src/EditorFeatures/Test/Utilities/BloomFilterTests.cs

```diff
@@ -139,6 +143,8 @@ public void TestSerialization2()
             using var reader = ObjectReader.TryGetReader(stream);
             var rehydratedFilter = BloomFilter.ReadFrom(reader);
             Assert.True(bloomFilter.IsEquivalent(rehydratedFilter));
+
+            GC.KeepAlive(keepAliveObjects);
```

Comment:
if you're going to update the bloomfilter tests in this way, i woudl actually prefer you then *add* an assertion that this set is empty.

Reply:
Empty and not empty are equally valid.

Reply:
teh same applies here.  when serializing, bloomfilters should not be adding any keepalive objects.  so if we're actually passing in this set, i would like the test to assert that that remains true.  if it were to ever change, i would want a test to fail immediately.

Reply:
➡️ Added trip-wire assertions with messages suggesting that the assertion is part of change detection as opposed to being part of behavior validation.